tests/integration/test_analysisd/test_syscollector/test_syscollector_integration.py

```diff
@@ -0,0 +1,128 @@
+receiver_sockets_params = [(ANALYSISD_QUEUE_SOCKET_PATH, 'AF_UNIX', 'UDP')]
+receiver_sockets = None
+alert_timeout = 5
+file_to_monitor = ALERT_FILE_PATH
+
+# ---------------------------------------- TEST_SYSCOLLECTOR_EVENTS -------------------------------------
+# Configuration and cases data
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_syscollector_integration.yaml')
+rule_file = "syscollector_rules.xml"
+
+# Enabled test configurations
+_, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+
+
+@pytest.mark.tier(level=2)
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=case_ids)
+def test_syscollector_integration(metadata, configure_local_internal_options_module, mock_agent_module,
+                                  configure_custom_rules, restart_analysisd, wait_for_analysisd_startup,
+                                  connect_to_sockets_function, file_monitoring):
+    """
+    description: Check if Analysisd handle Syscollector deltas properly by generating alerts.
+
+    wazuh_min_version: 4.4.0
+
+    tier: 2
+
+    parameters:
+        - metadata:
+            type: dict
+            brief: Get metadata from the module.
+        - mock_agent_module:
+            type: fixture
+            brief: Create mock agent and get agent_id
+        - configure_custom_rules:
+            type: fixture
+            brief: Copy custom rules to test.
+        - restart_analysisd:
+            type: fixture
+            brief: Restart analysisd daemon and truncate related log files.
+        - wait_for_analysisd_startup:
+            type: fixture
+            brief: Wait until analysisd is ready.
+        - connect_to_sockets_function:
+            type: fixture
+            brief: Connect to analysisd event queue.
+        - file_monitoring:
+            type: fixture
+            brief: Handle the monitoring of a specified file.
+
+    assertions:
+        - Verify that specific syscollector deltas trigger specific custom alert with certain values.
+
+    input_description:
+        Input dataset (defined as event_header + event_payload in cases_syscollector_integration.yaml)
+        cover, in most of the cases, INSERTED, MODIFIED and DELETED deltas
+        for each of the available scan; osinfo, hwinfo, processes, packages, network_interface,
+        network_address, network_protocol, ports and hotfixes.
+
+    expected_output:
+        Expected output (defined as alert_expected_values in cases_syscollector_integration.yaml)
+
+    tags:
+        - rules
+    """
+
+    # Get mock agent_id to create syscollector header
+    agent_id = mock_agent_module
+    event_header = f"d:[{agent_id}] {metadata['event_header']}"
+
+    # Add agent_id alert check
+    alert_expected_values = metadata['alert_expected_values']
+    alert_expected_values['agent.id'] = agent_id
+
+    # Create full message by header and payload concatenation
+    test_msg = event_header + metadata['event_payload']
+    print(test_msg)
```

Comment:
Please, remove it.

Reply:
Removed on commit 008183e5deff23cbd48756d2c63e1a91cb4c9dbc